include/diy/decomposition.hpp

```diff
@@ -540,14 +540,14 @@ struct Div
 template<class Bounds>
 void
 diy::RegularDecomposer<Bounds>::
-fill_divisions(std::vector<int>& divisions) const
+fill_divisions(std::vector<int>& p_divisions) const
```

Comment:
I'm not on board with the `p_` style. What does it accomplish? I'd like to avoid verbosity.